README.md

```diff
@@ -23,7 +23,7 @@ System Requirements
         # then install required packages
         sudo apt-get install git build-essential ncurses-dev fakeroot bc \
         u-boot-tools lzop flex bison libssl-dev \
-        gcc-linaro-arm-linux-gnueabihf-6.4
+        gcc-linaro-arm-linux-gnueabihf-6.4 gcc-arm-linux-gnueabihf-8.3
```

Comment:
Why do we need two compilers?

Reply:
The first one didn't work for me on Ubuntu 18.04. Happy to delete the first one, but not sure this would break things for others.

Reply:
Didn't work as in failed to install or failed to compile?

Reply:
Failed to compile. It said it couldn't find gcc-arm-linux-gnueabihf so I installed it.

But maybe it had to do with checking out stuff below, so I'll go and revert it.

Reply:
It isn't installed in a directory in `PATH`, but the build scripts should set `CROSS_COMPILE` to the correct path by default.

Reply:
I have a bunch of local mods so it is entirely possible that something is broken :smile:

Reply:
But I didn't succeed in the end (failed to link), so it's probably better not to merge. I think I'm going to close this and revisit it one day in an alternate universe.